### ISSUE TITLE
internet_service_id not persisted in terraform state

### ISSUE THREAD

**Issue by aloforte** [fortinetdev/terraform-provider-fortios#128]
Hi, I've a recurring issue with internet_service_id 
Env:
* terraform 0.14.2
* fortios 6.4.4 GA
* terraform fortios provider 1.6.18

Snippet:
``` terraform
resource "fortios_firewall_policy" "myrule" {
 name             = "my-rule"
  action           = "accept"
  schedule         = "always"
  comments         = "Teamcity-managed rule"
  logtraffic       = "utm"
  internet_service = "enable"

  dstintf {
    name = "external-iface"
  }
  internet_service_id {
    id = 393320
  }
  internet_service_id {
    id = 3604638
  }
  srcaddr {
    name = fortios_firewall_object_address.myserver.name
  }
  srcintf {
    name = "internal-iface"
  }
}
```

Results into rule being applied correctly on fortios side:
``` cfg
config firewall policy
    edit 190
        set name "my-rule"
        set uuid 709bf0c4-0498-51eb-cf05-58120ec1a545
        set srcintf "internal-iface"
        set dstintf "external-iface"
        set srcaddr "myserver"
        set internet-service enable
        set internet-service-name "Amazon-AWS" "GitHub-GitHub"
        set action accept
        set schedule "always"
        set utm-status enable
        set inspection-mode proxy
        set ssl-ssh-profile "certificate-inspection"
        set av-profile "default"
        set ips-sensor "default"
        set application-list "default"
        set comments "Teamcity-managed rule"
    next
end 
```

So rule is added and has the right internet_services listed.
Terraform status is never fully updated, resulting into missing internet_service_id sections, so on every subsequent apply the rule is "modified" again
``` 
  ~ resource "fortios_firewall_policy" "myrule" {
        id                          = "190"
        name                        = "my-rule"
        # (71 unchanged attributes hidden)


      + internet_service_id {
          + id = 393320
        }
      + internet_service_id {
          + id = 3604638
        }

        # (3 unchanged blocks hidden)
```

**Comment by frankshen01** [comment]
Hi @aloforte , Thanks for such a clear issue description, it can really help others save a lot of time. This problem occurs because 6.4.4 requires internet_service_name to configure. I will add support for this feature in the next version. It will be released in the next release (within a week). Thank you!

**Comment by frankshen01** [comment]
Supported and released, please see the latest version(v1.7.0): https://registry.terraform.io/providers/fortinetdev/fortios/latest and https://registry.terraform.io/providers/fortinetdev/fortios/latest/docs/resources/fortios_firewall_policy for details

**Validation:**
```
# cat main.tf
provider "fortios" {
  hostname = "192.168.52.177"
  insecure = "true"
}

resource "fortios_firewall_policy" "myrule" {
  name             = "my-rule1"
  action           = "accept"
  schedule         = "always"
  comments         = "Teamcity-managed rule"
  logtraffic       = "utm"
  internet_service = "enable"

  dstintf {
    name = "port1"
  }

  internet_service_name {
    name = "Amazon-AWS"
  }

  internet_service_name {
    name = "GitHub-GitHub"
  }

  srcaddr {
    name = "FABRIC_DEVICE"
  }

  srcintf {
    name = "port2"
  }
}

# terraform apply
....................

mysho (policy) # show
config firewall policy
    edit 1
        set name "my-rule1"
        set uuid 9f5e4edc-5116-51eb-d882-f0a016cc2b90
        set srcintf "port2"
        set dstintf "port1"
        set srcaddr "FABRIC_DEVICE"
        set internet-service enable
        set internet-service-name "Amazon-AWS" "GitHub-GitHub"
        set action accept
        set schedule "always"
        set comments "Teamcity-managed rule"
    next
end

# terraform plan
  ----
fortios_firewall_policy.myrule: Refreshing state... [id=1]

No changes. Infrastructure is up-to-date.

This means that Terraform did not detect any differences between your
configuration and real physical resources that exist. As a result, no
actions need to be performed.

```
 Thanks!

**Comment by aloforte** [comment]
Thank, you. It now works flawlessly.
